app/src/androidTest/java/mozilla/lockbox/Navigator.kt

```diff
@@ -91,6 +91,7 @@ class Navigator {
 
     fun gotoFxALogin() {
         welcome { tapGetStarted() }
+        welcome { tapSkipSecureYourDevice() }
```

Comment:
I wonder if we can add some logic to make this only tap `skip` when the device doesn't have any security... I think the `FingerprintStore.shared.isDeviceSecure` is available from the UI tests

Reply:
Oh I did not know that...I just tried and yup, looks like that works! I will update this PR with that change. That allows to enable two more existing tests :)